ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -48,6 +47,11 @@ export class Pay extends IronfishCommand {
       description:
         'The block sequence after which the transaction will be removed from the mempool. Set to 0 for no expiration.',
     }),
+    priority: Flags.string({
+      char: 'p',
+      description:
+        'The priority level for transaction fee estimation. The possible values are low, medium and high.',
```

Comment:
I think we may be able to add a param `options: ['low', 'medium', 'high']` to enforce this